integration/bmp/sl_cmd_test.go

```diff
@@ -35,15 +35,15 @@ var _ = Describe("`$bmp sl` integration tests", func() {
 
 		Context("executes --package-options", func() {
```

Comment:
So where is the negative tests now? Still important to have some if it's easy to do, as I think here.